plugin/backend.go

```diff
@@ -253,6 +267,45 @@ func (b *GcpAuthBackend) credentials(ctx context.Context, s logical.Storage) (*g
 	return creds.(*google.Credentials), nil
 }
 
+func (b *GcpAuthBackend) GetExternalAccountConfig(c *gcpConfig, ts *PluginIdentityTokenSupplier) *gcputil.ExternalAccountConfig {
+	b.Logger().Info("adding web identity token fetcher")
+	cfg := &gcputil.ExternalAccountConfig{
+		ServiceAccountEmail: c.ServiceAccountEmail,
+		Audience:            c.IdentityTokenAudience,
+		TTL:                 c.IdentityTokenTTL,
+		TokenSupplier:       ts,
+	}
+
+	return cfg
+}
+
+type PluginIdentityTokenSupplier struct {
+	sys      logical.SystemView
+	logger   hclog.Logger
+	audience string
+	ttl      time.Duration
+}
+
+var _ externalaccount.SubjectTokenSupplier = (*PluginIdentityTokenSupplier)(nil)
+
+func (p *PluginIdentityTokenSupplier) SubjectToken(ctx context.Context, opts externalaccount.SupplierOptions) (string, error) {
+	p.logger.Info("fetching new plugin identity token")
```

Comment:
nit: I would make this debug level or maybe even trace.

Reply:
Opted to keep this particular line at the Info level to be consistent with the [AWS](https://github.com/hashicorp/vault/blob/main/builtin/logical/aws/client.go#L157) and [Azure](https://github.com/hashicorp/vault-plugin-auth-azure/pull/151/files#diff-27d750705678a635de41deb5b02df7c6d93bb2a29af9c6e7c4591b487bca9f55R319) implementations